### DISCUSSION TITLE
Badges render strangely when prerending in mixed mode in a .NET 8 Blazor webapp.

### DISCUSSION THREAD

**Discussion by dimohy** [microsoft/fluentui-blazor#759] [Q&A]
The code you can test is in the link below.
https://github.com/dimohy/Blazor8Test

When I start with a .NET Blazor web app project template and apply mixed-mode (using both Server and Webassembly components) Fast Blazor, the badge displays strangely when pre-rendering. (It displays correctly again when the web assembly is loaded.)

How can I improve the strange behavior?

![image](https://github.com/microsoft/fluentui-blazor/assets/58620778/4da0d4a9-c3b2-4a73-a4de-9423aec79cd6)

After the Webassembly is all loaded

![image](https://github.com/microsoft/fluentui-blazor/assets/58620778/fc318297-7aeb-4001-9a62-aa8c1d04893a)

**Comment by vnbaaij** [comment]
Hi,

We have not done any work yet on making the library work with the new RenderModes so can't comment on why this is happening.
There might also be some issues due to Bootstrap still being in there.
